internal/tiltfile/k8s.go

```diff
@@ -139,16 +143,56 @@ func (s *tiltfileState) filterYaml(thread *starlark.Thread, fn *starlark.Builtin
 		return nil, err
 	}
 
-	match, rest, err := k8s.FilterByMetadataLabels(entities, metaLabels)
-	if err != nil {
-		return nil, err
+	var match, allRest []k8s.K8sEntity
+	match = append(match, entities...)
```

Comment:
is there a reason you copy entities into a new slice? maybe rename `entities` to `match`?

Reply:
...some vague sense that i shouldn't overwrite the `entities` slice? no real reason, no.